content/en/docs/pipelines/sdk/sdk-overview.md

```diff
@@ -107,6 +94,18 @@ The Kubeflow Pipelines SDK includes the following packages:
     represents a volume used to pass data between pipeline steps. `ContainerOp`s 
     can mount a `PipelineVolume` either via the constructor's argument 
     `pvolumes` or `add_pvolumes()` method.
+  * [`kfp.dsl.ParallelFor`](https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.dsl.html#kfp.dsl.ParallelFor)
+    represents a parallel for loop over a static or dynamic set of items in a pipeline.
+    Each iteration of the for loop will be executed in parallel.
+  
+  * [`kfp.dsl.ExitHandler`](https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.dsl.html#kfp.dsl.ExitHandler)
+    represents an exit handler that is invoked upon exiting a group of ops. A typical
+    usage of `ExitHandler` is garbage collection. Currently, one pipeline can
+    only include at most one `ExitHandler`.
+  
+  * [`kfp.dsl.Condition`](https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.dsl.html#kfp.dsl.ResourceOp)
+    represents a group of ops, that will only be executed when a certain condition is met.
+    The condition specified can be either determined at compile time or at runtime.
```

Comment:
If the condition is determined at compile time, is the condition still evaluated as part of the DAG? Or is it left out of the graph?

Reply:
Sorry I double checked it turns out that only boolean expression determined at runtime can be used here. Updated.